src/SFA.DAS.EmployerAccounts/Data/EmployerAgreementRepository.cs

```diff
@@ -168,5 +169,19 @@ public async Task SetAccountLegalEntityAgreementDetails(long accountLegalEntityI
             legalEntity.SignedAgreementVersion = signedAgreementVersion;
             await _db.Value.SaveChangesAsync();
         }
+
+        public async Task<AccountLegalEntity> GetOrganisationAgreement(long accountLegalEntityId)
```

Comment:
This method is called GetOrganisationAgreement but returns an AccountLegalEntity. Doesn't seem correct

Reply:
renamed the method